yml/dp02_dc2.yaml

```diff
@@ -8954,6 +8966,7 @@ tables:
     description: Average sky background.
     mysql:datatype: FLOAT
     fits:tunit: DN
+    ivoa:ucd: phot.flux.density.sb;instr.skyLevel
```

Comment:
Just `instr.skyLevel`, please -- this is in DN so it's not a "surface brightness", formally.